File: 백준/Bronze/28097. 모범생 포닉스/모범생 포닉스.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const n = Number(input.shift());
const t = input[0].split(' ').map(Number);
let d = 0;
let h = 0;
const sumTime = t.reduce((acc, curr) => acc + curr, 0);

d = Math.floor((sumTime + (n - 1) * 8) / 24);
h = (sumTime + (n - 1) * 8) % 24;

console.log(d, h);

File: 백준/Bronze/28097. 모범생 포닉스/모범생 포닉스.py
n = int(input())
t = list(map(int, input().split()))
d, h = 0, 0

d = (sum(t) + ((n - 1) * 8)) // 24
h = (sum(t) + ((n - 1) * 8)) % 24

print(d, h)